### PR TITLE
feat(store): integrate `store-node` submodule

### PR DESCRIPTION
**Pull request by sheli00**
Subtask of https://github.com/apache/incubator-hugegraph/issues/2265 and https://github.com/apache/incubator-hugegraph/issues/2482

Detailed doc can be seen in [this link](https://hugegraph.feishu.cn/wiki/TGc7wxTZsiki3skxvTHcdyT0n1e)

### CODE REVIEW

File: hugegraph-store/hg-store-node/pom.xml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                <exclusion>
+                    <groupId>io.grpc</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hugegraph</groupId>
+            <artifactId>hg-store-core</artifactId>
+        </dependency>
```
Comment (VGalaxies) [inline hugegraph-store/hg-store-node/pom.xml:123]: Consider prioritizing the integration of the `store-core` module?

**Comment by sheli00** [comment]
@VGalaxies @Pengzna PTAL

**Comment by VGalaxies** [comment]
@sheli00 could reformat the code for store-node before merging

image

**Comment by sheli00** [comment]
> @sheli00 could reformat the code for store-node before merging
> 
> image



> @sheli00 could reformat the code for store-node before merging
> 
> image

Comparing the difference between the code before and after formatting, there should be no major problems

**Review by VGalaxies — APPROVED** [review]

**Review by VGalaxies — APPROVED** [review]

File: hugegraph-store/hg-store-node/banner.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+  _   _  ____      ____ _____ ___  ____  _____      _   _  ___  ____  _____
+ | | | |/ ___|    / ___|_   _/ _ \|  _ \| ____|    | \ | |/ _ \|  _ \| ____|
+ | |_| | |  _ ____\___ \ | || | | | |_) |  _| _____|  \| | | | | | | |  _|
+ |  _  | |_| |_____|__) || || |_| |  _ <| |__|_____| |\  | |_| | |_| | |___
+ |_| |_|\____|    |____/ |_| \___/|_| \_\_____|    |_| \_|\___/|____/|_____|
```
Comment (imbajin) [inline hugegraph-store/hg-store-node/banner.txt:5]:
```suggestion
 |_| |_|\____|    |____/ |_| \___/|_| \_\_____|    |_| \_|\___/|____/|_____|

```

**Review by imbajin — APPROVED** [review]

**Review by VGalaxies — APPROVED** [review]

**Pull request merged**